climada/util/forecast.py

```diff
@@ -56,3 +56,35 @@ def __init__(
         )
         self.member = np.asarray(member) if member is not None else np.array([])
         super().__init__(**kwargs)
+
+    def idx_member(self, member: np.ndarray) -> np.ndarray:
+        """Return boolean array where self.member == member using numpy.isin()
+
+        Parameters
+        ----------
+        member : np.ndarray
+            Forecast ensemble members, given as integers.
```

Comment:
That is not very clear. How about
```suggestion
            Index of the forecast members in the forecast.member array.
```

Reply:
See https://github.com/CLIMADA-project/climada_python/pull/1183/commits/12bf2b620a2c7cd46d5d34afaf13424ef448a092